package.json

```
@@ -35,7 +35,7 @@
     "dev": "webpack serve --config webpack/webpack.config.js --env env=development --open",
     "start": "node server.js",
     "build": "webpack --config webpack/webpack.config.js --env env=production",
-    "lint": "eslint src/**/*.{js,ts,tsx}",
+    "lint": "eslint 'src/**/*.{js,ts,tsx}'",
```

There's a subtle issue: on windows single quotes don't work as expected.
Therefore I suggest this instead:
```suggestion
    "lint": "eslint \"src/**/*.{js,ts,tsx}\"",
```

(we do a similar thing in the profiler for stylelint... but not eslint curiously)

---

done